docs/sdks/source-sdk/send-api.md

```diff
@@ -55,6 +55,7 @@ This section lists the parameters you must send for all actions and specifies wh
 | Threads | create |
 | Users | create |
 | Status | mark_as_read |
+| Typing | start stop |
```

Comment:
Maybe a line break between the 2 would be clearer? They look too close in your screenshot

Reply:
good idea, I tried with a comma at first but it wasn't great, it's way better with a line break :+1:

done in c455295